testing/spectypes/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@prysm//tools/go:def.bzl", "go_library")
```

Comment:
Curious, is there any reason the `core` package couldn't be used ?  A lot of the previous state related functions were added in there in your previous PRs

Reply:
An out-of-band discussion led us to moving this file to `testing/utils` for now, because a number of similar "populate a struct for testing" functions live there.